client/app/components/ProfileCard.jsx

```
@@ -281,19 +317,56 @@ export default function ProfileCard({ profile, aaWalletAddress }) {
-                  style={{ marginBottom: "16px" }}
+                  style={{ marginBottom: "0.5em" }}
                 />
+                <Space wrap style={{ marginBottom: "0.5em" }}>
+                  {suggestedTips.map((tip) => (
+                    <Button
+                      size="small"
+                      type={tipAmount === tip.value ? "primary" : "default"}
+                      onClick={() => setTipAmount(tip.value)}
+                      style={{ minWidth: 60 }}
+                    >
+                      {tip.label}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)